custom_types.yaml

```diff
@@ -587,12 +587,12 @@ node_types:
     interfaces:
       Standard:
         configure:
-          implementation: mesos/mesos_master_install.yml
+          implementation: artifacts/mesos/mesos_master_install.yml
           inputs:
-            consul_server_ips: { get_attribute: [ HOST, public_address ] }
-            zookeeper_host_list_ips: { get_attribute: [ HOST, public_address ] }
-            zookeeper_peers_ips: { get_attribute: [ HOST, public_address ] }
-            mesos_masters_list_ips: { get_attribute: [ HOST, public_address ] }
+            consul_server_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            zookeeper_host_list_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            zookeeper_peers_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            mesos_masters_list_ips: [ { get_attribute: [ HOST, public_address ] } ]
```

Comment:
I can do that as a workaround for now, but this is too indigo specific to make it upstream. I'll do that for now but I still think it would be good to change that in a later release.